automation_testing/pages/BasePage.ts

```diff
@@ -118,4 +118,19 @@ export class BasePage {
 		await this.page.keyboard.up("Shift");
 		await this.page.keyboard.press("Backspace");
 	}
+
+	async validateFontWeightFromElement(element: Locator, expectedValue: string): Promise<void> {
+		const elementFontWeight = await ((element).evaluate(el => getComputedStyle(el).fontWeight));
+		expect(elementFontWeight).toBe(expectedValue);
+	}
+
+	async validateMarginValueFromElement(element: Locator, expectedValue: string, isRight: boolean): Promise<void> {
+		let elementMargin: string;
+		if (isRight) {
+			elementMargin = await ((element).evaluate(el => getComputedStyle(el).marginLeft));
```

Comment:
I think the conditions might be swapped. If isRight is true shouldn't it be testing for the marginRight, and marginLeft when false?

Reply:
I think the condition is not wrong. If the icon is on the left side of the button, the margin with the text will be on the right side of the icon. That's why I wrote the condition to be that way.